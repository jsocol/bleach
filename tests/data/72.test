<form action="javasc
ript:alert(1)"><button>XXX</button></form>
