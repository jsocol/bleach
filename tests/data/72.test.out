<form><button>XXX</button></form>
